node/src/service.rs

```diff
@@ -214,7 +225,7 @@ where
 
 	// Spawn Frontier EthFilterApi maintenance task.
 	if filter_pool.is_some() {
-		use futures::StreamExt;
+		// use futures::StreamExt;
```

Comment:
Do we need to keep as comment ?

Reply:
@tgmichel what do you think?

I commented this because I needed the export elsewhere. But I remember you wanted to keep this self contained. Hopefully this will be obsolete soon anyway with your upcoming changes to this section.

Reply:
Actually, you're right. I'll delete it. 
```suggestion
```